content/en/blog/falcosidekick-reponse-engine-part-4-tekton.md

```diff
@@ -0,0 +1,613 @@
+---
+title: "Kubernetes Response Engine, Part 4: Falcosidekick + Tekton"
+date: 2021-04-11
+author: Edvin Norling
+slug: falcosidekick-reponse-engine-part-4-tekton
+---
+
+> *This blog post is part of a series of articles about how to create a `Kubernetes` response engine with `Falco`, `Falcosidekick` and a `FaaS`.*
+>
+> See other posts:
+>
+> - [Kubernetes Response Engine, Part 1 : Falcosidekick + Kubeless]({{< ref "/blog/falcosidekick-reponse-engine-part-1-kubeless" >}})
+> - [Kubernetes Response Engine, Part 2 : Falcosidekick + OpenFaas]({{< ref "/blog/falcosidekick-reponse-engine-part-2-openfass" >}})
```

Comment:
```suggestion
> - [Kubernetes Response Engine, Part 2 : Falcosidekick + OpenFaas]({{< ref "/blog/falcosidekick-reponse-engine-part-2-openfass" >}})
> - [Kubernetes Response Engine, Part 3 : Falcosidekick + Knative]({{< ref "/blog/falcosidekick-reponse-engine-part-3-knative" >}})
```

Reply:
Didn't see that you fixed it as well, so thats why I didn't use yours.
Thanks